### ISSUE TITLE
This Error

### ISSUE THREAD

**Issue by Hikariett** [infinitefusion/infinitefusion-e18#106]
![PKIF error](https://github.com/infinitefusion/infinitefusion-e18/assets/39041837/a3e7c166-a6bb-428f-8dd4-41617adf432a)

I keep getting the error in the screenshot when trying to load the game.

**Comment by eddperpich** [comment]
Have you tried redownloading the game?

**Comment by Hikariett** [comment]
Yes I have, multiple times infact - I managed to "fix it" by removing my older save and the game seems to load fine now... not sure how long that will last tho
